README.md

```diff
@@ -5,15 +5,16 @@ The Flink CDC Connectors integrates Debezium as the engine to capture data chang
 
 This README is meant as a brief walkthrough on the core features with Flink CDC Connectors. For a fully detailed documentation, please see [Documentation](https://ververica.github.io/flink-cdc-connectors/master/).
 
-## Supported (Tested) Connectors
-
-| Database | Version |
-| --- | --- |
-| MySQL | Database: 5.7, 8.0.x <br/>JDBC Driver: 8.0.16 |
-| PostgreSQL | Database: 9.6, 10, 11, 12 <br/>JDBC Driver: 42.2.12|
-| MongoDB | Database: 3.6, 4.x, 5.0 <br/>MongoDB Driver: 4.3.1|
-| Oracle | Database: 11, 12, 19 <br/>Oracle Driver: 19.3.0.0|
-| Sqlserver | Database: 2017, 2019 <br/>JDBC Driver: 7.2.2.jre8|
+## Supported (Tested) Databases
+
+| Protocol   | Connector                                                                                                      | Database                                                                                                                                                                                                                                                                                                                                                                           | Driver                  |
+|------------|----------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------|
+| MySQL      | [mysql-cdc](https://ververica.github.io/flink-cdc-connectors/master/content/connectors/mysql-cdc.html)         | <li> [MySQL](https://dev.mysql.com/doc): 5.7, 8.0.x <li> [MariaDB](https://mariadb.org): 10.x <li> [PolarDB MySQL](https://www.aliyun.com/product/polardb): 5.7, 8.0.x <li> [PolarDB X](https://github.com/ApsaraDB/galaxysql): 2.0.1 <li> [RDS MySQL](https://www.aliyun.com/product/rds/mysql): 5.7, 8.0.x <li> [Aurora MySQL](https://aws.amazon.com/cn/rds/aurora): 5.7, 8.0.x | JDBC Driver: 8.0.16     |
+| PostgreSQL | [postgresql-cdc](https://ververica.github.io/flink-cdc-connectors/master/content/connectors/postgres-cdc.html) | <li> [PostgreSQL](https://www.postgresql.org): 9.6, 10, 11, 12                                                                                                                                                                                                                                                                                                                     | JDBC Driver: 42.2.12    |
```

Comment:
when we cut branch release-2.2 from master, we don't want to change the link, could you use `[postgresql-cdc](postgres-cdc.md)` ?